ui-tests/src/test/java/io/syndesis/qe/pages/customizations/extensions/TechExtensionDetailPage.java

```diff
@@ -0,0 +1,31 @@
+package io.syndesis.qe.pages.customizations.extensions;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+
+public class TechExtensionDetailPage extends SyndesisPageObject {
+
+	private static final class Element {
+		public static final By ROOT = By.cssSelector("syndesis-tech-extension-detail");
+	}
+
+	@Override
+	public SelenideElement getRootElement() {
+		return $(Element.ROOT).shouldBe(visible);
+	}
+
+	@Override
+	public boolean validate() {
+		//TODO Deeper validation
+		this.getButton("Update").shouldBe(visible);
+		this.getButton("Delete").shouldBe(visible);
+		
+		return true;
```

Comment:
This's also kind of strange `validate()` method :)

Reply:
Same here.